containers/graalvm-build/docker/root/scripts/setup-users.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xeuo pipefail
+
+user_ids="1000 1007"
```

Comment:
Do we need this for the building GraalVM? Why can't the current user be used when running the docker image?

Reply:
Let's keep this file. The scripts were written assuming that the group id 1000 exists.